src/ConcreteOperations/interior.jl

```diff
@@ -0,0 +1,28 @@
+export is_interior_point
+
+
+"""
+    is_interior_point(d::AbstractVector, P::LazySet; p=Inf, ε=_TOL_F64.rtol)
+
+Check if the point `d` is contained in the interior of the convex set `P`.
+
+### Input
+
+- `d`  -- coordinate of point
+- `P`  -- lazy set
+- `p`  -- norm of the Ball used to apply the error tolerance
+- `ε`  -- error tolerance of check
+
+### Output
+
+Boolean which indicates if point `d` is contained in `P`.
+
+### Algorithm
+
+The implementation checks if a `BallInf` with center `d` and radius `ε` is
+contained in the set `P` which results in a numerical check for `d ∈ interior(P)`
+with error tolerance `ε` with default value `_TOL_F64.rtol`.
```

Comment:
```suggestion
with error tolerance `ε`.
```
I would not talk about default values in the documentation.